config.py

```diff
@@ -0,0 +1,12 @@
+import argparse
+
+def arg_parse():
+    parser = argparse.ArgumentParser(description='PyTorch DQN Training')
```

Comment:
Can you please replace all single-quotation-marks to double-quotation-marks (since all other parts of this repo use the latter)? (However, though this is recommended, it is not necessarily necessary.)